modules/ballerina-native/src/main/ballerina/ballerina/lang/files/natives.bal

```diff
@@ -0,0 +1,41 @@
+package ballerina.lang.files;
+
+import ballerina.doc;
+
+@doc:Description { value:"Closes a given file and its stream"}
+@doc:Param { value:"file: The File struct" }
+native function close (File file);
+
+@doc:Description { value:"Gets the stream from a local file"}
+@doc:Param { value:"file: The file struct" }
+native function open (File file, string accessMode);
+
+@doc:Description { value:"Get blob value of a file"}
```

Comment:
Can you please follow the same pattern when documenting functions? Please have a look at other native functions. 
here s/Get/Gets/g